salt/metalk8s/internal/preflight/mandatory.sls

```diff
@@ -38,8 +38,22 @@ Set sysctl {{ item }} value to {{ value }}:
     - require:
       - kmod: Add the module br_netfilter to kernel
     {%- endif %}
+    - require_in:
+      - cmd: Apply sysctl configuration files
 {%- endfor %}
 
+{#- This is needed to ensure values loaded from files match in-memory ones. #}
```

Comment:
Minor: We do not really care, but here it's a comment about YAML not jinja so to me it should be a YAML comment :smiley:

Reply:
done